pages/linux/objcopy.md

```diff
@@ -0,0 +1,24 @@
+# objcopy
+
+> Copy the contents of an object file to another.
+> More information: <https://manned.org/objcopy>.
+
+- Copy data to another file:
+
+`objcopy {{path/to/source_file}} {{path/to/target_file}}`
+
+- Translate object files from one format to another:
+
+`objcopy --input-target={{inputformat}} --output-target={{output_format}} {{path/to/source_file}} {{path/to/target_file}}`
+
+- Strip all symbol information from file:
+
+`objcopy --strip-all {{path/to/source_file}} {{path/to/target_file}}`
+
+- Strip debugging information from file:
+
+`objcopy --strip-debug {{path/to/source_file}} {{path/to/target_file}}`
+
+- Copy a specific section from the source to destination:
+
+`objcopy --only-section={{section}} {{path/to/source_file}} {{path/to/target_file}}`
```

Comment:
```suggestion
`objcopy --only-section {{section}} {{path/to/source_file}} {{path/to/target_file}}`
```
Checked the docs locally and `=` here isn't required for the input to work.

Ref.

```bash
kbdharun@server:~# objcopy -h
Usage: objcopy [option(s)] in-file [out-file]
 Copies a binary file, possibly transforming it in the process
 The options are:
  -I --input-target <bfdname>      Assume input file is in format <bfdname>
  -O --output-target <bfdname>     Create an output file in format <bfdname>
  -B --binary-architecture <arch>  Set output arch, when input is arch-less
  -F --target <bfdname>            Set both input and output format to <bfdname>
     --debugging                   Convert debugging information, if possible
  -p --preserve-dates              Copy modified/access timestamps to the output
  -D --enable-deterministic-archives
                                   Produce deterministic output when stripping archives (default)
  -U --disable-deterministic-archives
                                   Disable -D behavior
  -j --only-section <name>         Only copy section <name> into the output
     --add-gnu-debuglink=<file>    Add section .gnu_debuglink linking to <file>
... (The below options aren't relevant for this review comment)
```